src/managing-workflow/configuring-dns.md

```diff
@@ -1,67 +1,71 @@
 
-You can find the IP addresses of your Kubernetes cluster nodes by
-running `kubectl get nodes`.
+* The controller should be accessible at `deis.example.com`
 
-## Necessary DNS records
+* Applications should be accessible (by default) at `<application name>.example.com`
 
-Deis requires a wildcard DNS record. Assuming `myapps.com` is the top-level domain
-apps will live under:
+Given that this is the case, the primary objective in configuring DNS is that traffic for all subdomains of a cluster's domain be directed to the cluster node(s) hosting the platform's router component, which is capable of directing traffic within the cluster to the correct endpoints.
 
-* `*.myapps.com` should have "A" record entries for each of the load balancer's IP addresses
 
-Apps can then be accessed by browsers at `appname.myapps.com`, and the controller will be available to the Deis client at `deis.myapps.com`.
+## With a Load Balancer
 
-[AWS recommends][] against creating "A" record entries; instead, create a wildcard "CNAME" record entry for the load balancer's DNS name, or use Amazon [Route 53][].
+Generally, it is recommended that a [load balancer][] be used to direct inbound traffic to one or more routers.  In such a case, configuring DNS is as simple as defining a wildcard record in DNS that points to the load balancer.
 
-These records are necessary for all deployments of Deis other than Vagrant clusters.
+For example, assuming a domain of `example.com`:
 
-## DNS without a Load Balancer
+* An `A` record enumerating each of your load balancer(s) IPs (i.e. DNS round-robining)
 
-On some platforms (Vagrant, for instance), a load balancer is not an easy or practical thing to
-provision. In these cases, one can directly identify the public IP of a Kubernetes node that is
-hosting a router pod and use that information to configure DNS or the local `/etc/hosts` file.
+* A `CNAME` record referencing an existing fully-qualified domain name for the load balancer
+    * Per [AWS' own documentation][AWS recommends], this is the recommended strategy when using AWS Elastic Load Balancers, as ELB IPs may change over time.
 
-You can find the IP address of a node using `kubectl`:
+DNS for any applications using a "custom domain" (a fully-qualified domain name that is not a subdomain of the cluster's own domain) can be configured by creating a `CNAME` record that references the wildcard record described above.
+
+Although it is dependent upon your distribution of Kubernetes and your underlying infrastructure, in many cases, the IP(s) or existing fully-qualified domain name of a load balancer can be determined directly using the `kubectl` tool:
 
 ```
-$ kubectl get pods --namespace=deis | grep deis-router
-deis-router-ih25q           1/1       Running   0          2h
+$ kubectl describe service deis-router --namespace=deis
```

Comment:
Maybe add a ` | grep LoadBalancer Ingress` after this, to make it more apparent to users what to look for?

Also, I think it would help to say that this field won't exist until after the LB has actually been created. I've gotten caught up on that before, wondering why there's no LB when in fact it's still coming up...